### PR TITLE
Add dca guide

### PR DESCRIPTION
**Pull request by benlvb**

### CODE REVIEW

File: guides/10-dca.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+- On Day 1, you will receive $300 USDC worth of SOL in your wallet automatically. If the price of SOL is $30 USDC / SOL, you will receive 29.97 SOL *(net of fees)* in your wallet.
+- On Day 2, you will receive $300 USDC worth of SOL in your wallet. If the price of SOL is $25 USDC / SOL, you will receive 11.988 SOL *(net of fees)* in your wallet. 
+- On Day 3, you will receive $300 USDC worth of SOL in your wallet. If the price of SOL stay stagnant and remain as $25 USDC / SOL, you will receive 11.988 SOL *(net of fees)* in your wallet.
+
+:::tip Caveat to auto-withdrawal
+In the event your desired purchased token is not SOL, auto withdrawal for each order will only work if user have the correct associated token account (ATA) open and ready.
+:::
+
+- By default, Jupiter DCA open the necessary ATA for your wallet when you create a Jupiter DCA account. However, if you close your purchased token's ATA manually via wallet interface or any other 3rd-party tool, tokens will not transferred to you automatically on every order but only at the end of your DCA cycles as a single lumpsum amount. **This only applicable to SPL tokens and not SOL.**
```
Comment (Timoon21) [inline guides/10-dca.md:132]:
```suggestion
- By default, Jupiter DCA open the necessary ATA for your wallet when you create a Jupiter DCA account. However, if you close your purchased token's ATA manually via wallet interface or any other 3rd-party tool, tokens will not transferred to you automatically on every order but only at the end of your DCA cycles as a single lumpsum amount. **This only applicable to SPL tokens. For SOL: Jupiter takes care to open/close your wrapped account.**
```
To clarify what happens in the SOL scenario.

File: guides/10-dca.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+By default, if your wallet’s ATA remains open, your purchase tokens are automatically transferred to you on every order (the program only allows your tokens to be sent to your wallet and no one else’s). 
+
+However, in the event, there are remaining purchase tokens yet to be transferred to you (ie. if you close your wallet’s token account for your purchase token halfway through the DCA cycle as described in auto-withdrawal section), the DCA program will close the PDA in-token account and utilize the rent recovered to open the correct ATA for your wallet to store your purchased token. 
+
+This allows the program to transfer the remaining purchased tokens to you. As such, you would only receive 2/3 of the rent incurred to initialise your DCA account. Having said that, this 1/3 rent is not absorbed by any third-party. This rent will remain recoverable by you if you decide to close your ATA that “holds” your purchased token. 
+
+:::warning 
+Do not close your ATA without first withdrawing the tokens or swapping them for other tokens. Doing so can result in the lost of tokens. This is not unique to Jupiter or Jupiter’s DCA. This has to do with how Solana wallets and accounts work.
```
Comment (Timoon21) [inline guides/10-dca.md:153]: Taking account of the above, in what cases is a loss of token possible?

File: guides/10-dca.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+:::info Auto Close
+At the end of your DCA orders, you would not need to perform any actions, any remaining tokens and rent is transferred to your wallet account directly.
+:::
+
+By default, if your wallet’s ATA remains open, your purchase tokens are automatically transferred to you on every order (the program only allows your tokens to be sent to your wallet and no one else’s). 
+
+However, in the event, there are remaining purchase tokens yet to be transferred to you (ie. if you close your wallet’s token account for your purchase token halfway through the DCA cycle as described in auto-withdrawal section), the DCA program will close the PDA in-token account and utilize the rent recovered to open the correct ATA for your wallet to store your purchased token. 
+
+This allows the program to transfer the remaining purchased tokens to you. As such, you would only receive 2/3 of the rent incurred to initialise your DCA account. Having said that, this 1/3 rent is not absorbed by any third-party. This rent will remain recoverable by you if you decide to close your ATA that “holds” your purchased token.
```
Comment (Timoon21) [inline guides/10-dca.md:150]:
```suggestion
This allows the program to transfer the remaining purchased tokens to you. 
```
Not sure if this part is needed. If we're talking about the wallet's ATA, it's well described before it was taken from the rent PDA. So obviously, it's not returned at the end of the DCA but used in the wallet's ATA.

File: guides/10-dca.md (hunk 2 above)
Comment (joshteng) [reply guides/10-dca.md:155]: @Timoon21 Won't happen. It's just a matter of when they receive the tokens - after every single order or at the very end.

**Review by joshteng — APPROVED** [review]

File: guides/10-dca.md (hunk 3 above)
Comment (joshteng) [reply guides/10-dca.md]: This part is actually more about rent refund for the accounts created which includes the ATAs as well as the DCA program account. In certain cases, they would get rent refunded for 3 accounts and sometimes just 2 out of the 3.

We can remove, I don't think most people would dive into the details of rent I guess

**Pull request merged**